netlify.toml

```
@@ -25,25 +25,31 @@
 [[redirects]]
   from = "/releases/*"
   to = "https://releases.nixos.org/:splat"
-  status = 301
+  status = 302
   force = true
 
 [[redirects]]
   from  = "/nix/install"
   to = "https://releases.nixos.org/nix/nix-2.3.3/install"
   status = 200
   #to = "https://channels.nixos.org/nix-latest/install"
-  #status = 301
+  #status = 302
   force = true
 
 [[redirects]]
   from  = "/hydra"
   to = "https://github.com/NixOS/hydra"
-  status = 301
+  status = 302
   force = true
 
 [[redirects]]
   from  = "/nixops"
   to = "https://github.com/NixOS/nixops"
-  status = 301
+  status = 302
+  force = true
+
+[[redirects]]
+  from  = "/nixops"
+  to = "https://github.com/NixOS/nixops/manual"
```

this is a non working link, you probably mean:

```suggestion
  to = "https://github.com/NixOS/nixops/tree/master/doc/manual"
```

---

The rendered manual is at https://hydra.nixos.org/job/nixops/master/tarball/latest/download-by-type/doc/manual.